packages/core/contracts/financial-templates/optimistic-distributor/OptimisticDistributor.sol

```diff
@@ -0,0 +1,420 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
comment about location: does it make sence for this to be in the `financial-templates` directory? we might want to move both the `OR` and the `OD` into their own packages along with the `Merkle-distributor`. 

dont need to do anything here for this @Reinis-FRP just wanted to get feedback on this from @nicholaspai & @mrice32

Reply:
Yeah I think a follow up PR should move both the `OR` and the `OD` into their own folder but for now keeping it here is consistent with where the `OR` is located. Do you think we should move them in this PR?

Reply:
na let's move it separately in one PR change.